sdgym/benchmark.py

```diff
@@ -551,7 +553,11 @@ def _create_sdgym_script(params, output_filepath):
     # Generate the output script to run on the e2 instance
     synthesizer_string = 'synthesizers=['
     for synthesizer in params['synthesizers']:
-        synthesizer_string += synthesizer.__name__ + ', '
+        if isinstance(synthesizer, str):
+            synthesizer_string += synthesizer + ', '
+        else:
+            synthesizer_string += synthesizer.__name__ + ', '
+    synthesizer_string = synthesizer_string[:-2]
```

Comment:
Just added this to clean up to remove trailing `, ` from script.

Reply:
Could this mess up if the synthesizers list is empty? Because then synthesizer_string[:-2] would be 'synthesizers' since it would clip '=]' right?

Reply:
Yep fixed it with a check for empty parameters